src/main/java/de/retest/web/selenium/TestHealer.java

```diff
@@ -152,10 +141,25 @@ private WebElement findElementByLinkText( final ByLinkText by ) {
 	}
 
 	private WebElement findElementByCssSelector( final ByCssSelector by ) {
-		final String selector = retrieveUsableCssSelector( by );
-
-		final Element actualElement = de.retest.web.selenium.By.findElementByAttribute( lastExpectedState,
-				lastActualState, CLASS, value -> ((String) value).contains( selector ) );
+		final String selector = ByWhisperer.retrieveCssSelector( by );
+		if ( selector.matches( ".*[<>:+\\s\"\\[\\*].*" ) ) {
+			throw new IllegalArgumentException(
+					"For now, only simple CSS selectors are implemented. Please report your chosen selector ('"
+							+ selector + "') at https://github.com/retest/recheck-web/issues." );
```

Comment:
Since this will be some time until this will be implemented, should we just `log.warn` and return `null` so to say that no element has been found? This would fix the breakage while ensuring the message.

The same goes for the XPath below.

Reply:
That's the best idea. Obviously much better! Thanks